react/global.css

```diff
@@ -44,8 +46,8 @@
 
 @media screen and (min-width: 80em) {
   .vtex-footer__container {
-    padding-left: 8.75rem;
-    padding-right: 8.75rem;
+    padding-left: 1.5rem;
+    padding-right: 1.5rem;
   }
```

Comment:
I think that, besides `20px`'s value, all of them are achievable with Tachyons. Can you gather with @oldufre for this to happen? And also, using `px` is not encouraged.

Reply:
I've replaced the css by using tachyons and switched the px sizes by it's equivalent in rem.

Reply:
Can't you move the rest of CSS to tachyons classes? :smile: 

Please let me know if it will be possible